CryptoTechReminderSystem.AcceptanceTest/ApiEndpointResponse/HarvestProject1UserAssignmentsResponse.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Add test data for the Harvest `user_assignments` endpoint.

Harvest uses `user_assignments` as the join table in the many-to-many relationship between Users and Projects, docs [here](https://help.getharvest.com/api-v2/users-api/users/project-assignments/).

The `is_project_manager` flag of the user assignment denotes if the User is a Project Manager of a Project.